docs/api/web-contents.md

```diff
@@ -1034,6 +1034,8 @@ In the browser window some HTML APIs like `requestFullScreen` can only be
 invoked by a gesture from the user. Setting `userGesture` to `true` will remove
 this limitation.
 
+Code execution will be suspended until web page stop loading.
```

Comment:
It should "finish loading" instead of "stop loading".

```suggestion
If web page is still loading, code execution will be suspended until loading is finished.
```

Reply:
Thank for your attention, IMHO I still think "stop loading" is better because before page has finished loading , we could use "webContents.stop()" imperatively to stop the loading and run code supplied to "contents.executeJavaScript()" immediately. 

In the meantime I'm happy to accept your final modification.  Any reminder is better than nothing.

Reply:
Yeah you are right, after checking the code I found it is waiting for "did-stop-loading", which is "stop loading".